modules/core/src/main/scala/com.snowplowanalytics/iglu.schemaddl/jsonschema/mutate/Mutate.scala

```diff
@@ -0,0 +1,286 @@
+/*
+ * Copyright (c) 2014-2022 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.iglu.schemaddl.jsonschema.mutate
+
+import com.snowplowanalytics.iglu.schemaddl.jsonschema.Schema
+import com.snowplowanalytics.iglu.schemaddl.jsonschema.properties._
+
+object Mutate {
```

Comment:
Should this object also be private like `Narrowed` and `Widened`? But not in `mutate` package but e.g. `schemaddl`?

Reply:
Good point.  Originally I left it public, because I was thinking that rdb-transformer would need to mutate the schema.  Later I changed the schema-ddl parquet implementation to always call mutate, so it doesn't need to be public anymore.

I partly think all of these mutate functions could be useful beyond just schema-ddl, and that makes me want them to be public.  On the other hand, if we make them public then we need to support them, and I prefer it if schema-ddl has a small opinionated API interface.